### PR TITLE
T-REX 1.1

### PR DESCRIPTION
**Pull request by Myna65**

### CODE REVIEW

File: contracts/registry/IdentityRegistry.sol [hunk 1]
```diff
@@ -1,40 +1,73 @@
...
-    //Array storing trusted claim types of the security token.
-    uint256[] claimTypes;
 
-    IdentityContract[] public identities;
+    /**
+     * @dev Returns true if the caller is the current owner.
+     */
+    function isAgent() public view returns (bool) {
+        return agents[msg.sender];
+        // return msg.sender == _agent;
```
Comment (Myna65) [inline contracts/registry/IdentityRegistry.sol]:
```suggestion
```

File: contracts/token/IToken.sol [hunk 2]
```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.5.10;
+
+//interface
+contract IToken{
+
+    string public name;
+    string public symbol;
+    uint8 public constant decimals = 0;
```
Comment (Joachim-Lebrun) [inline contracts/token/IToken.sol]: i don't think we should set a value for this on the interface, it should be set on the implementation side.

Comment (Joachim-Lebrun) [reply contracts/token/IToken.sol]: https://github.com/TokenySolutions/T-REX/pull/17

**Comment by Joachim-Lebrun** [comment]
@dilipror @Nakasar @Aboudjem @Myna65 @lfalempin @adetante @fcroiseaux  we expect to merge soon, if you can review the code it would be great :)

**Pull request merged**